<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnOpenPort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Port</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lstMessages">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>370</y>
      <width>81</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="lstA1">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>370</y>
      <width>81</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="lstA2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>370</y>
      <width>81</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="lstA3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>370</y>
      <width>81</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblSend">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>300</y>
      <width>49</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblGot">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>300</y>
      <width>49</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customplot" native="true">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>801</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>191</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>309</width>
       <height>189</height>
      </rect>
     </property>
     <widget class="QPushButton" name="pushButtonAngles">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Angles</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAngularSpeeds">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>291</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Angular Speeds</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonLinearAccelerations">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>291</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Linear Accelerations</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonLinearSpeeds">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>291</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Linear Speeds</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonLinearCoordinates">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>291</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Linear Coordinates</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonDEBUGRequest">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>291</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>DEBUG Request</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxRequest">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>231</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>                      Request Settings</string>
    </property>
    <widget class="QPushButton" name="btnStart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStop">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditDelay">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxAngles">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Angles</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxAngularSpeeds">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Angular Speeds</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxLinearAccelerations">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Linear Accelerations</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxLinearSpeeds">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Linear Speeds</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxLinearCoordinates">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Linear Coordinates</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSendCloseRequest">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxSettings">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>231</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>                      Command Settings</string>
    </property>
    <widget class="QLineEdit" name="lineEditA1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxAngles_cmd">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Angles</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxAngularSpeeds_cmd">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Angular Speeds</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxLinearAccelerations_cmd">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Linear Accelerations</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxLinearSpeeds_cmd">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Linear Speeds</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxLinearCoordinates_cmd">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Linear Coordinates</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditA2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditA3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSend">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSendCloseCommand">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelBufferL">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>370</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxCmdAdd">
    <property name="geometry">
     <rect>
      <x>889</x>
      <y>370</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>New cmd params</string>
    </property>
    <widget class="QLineEdit" name="lineEditName">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditParamsN">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditIdAlias">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditIdNumber">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Class Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Parameters n</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID Alias</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID number</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAddCMD">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
